templates/publisher/register-name.html

```diff
@@ -10,6 +10,11 @@
   <div class="row">
     <h1>Register a snap name</h1>
```

Comment:
`<h1>` seems too large comparing to other headings used around snapcraft.io

'My snaps' main heading uses: `<h1 class="p-heading--three">My snaps</h1>`, so I guess we should use same sizing here.